styleguide/content/utils/page-view.js

```diff
@@ -0,0 +1,11 @@
+import ReactGA from 'react-ga';
+
+ReactGA.initialize('UA-7529271-29');
```

Comment:
Per our discussion, I need to generate a new GA site/number.

Also, could perhaps look at setting it based off an ENV.

Reply:
@roblevintennis got a new number for ya:

```
UA-7529271-31
```

Reply:
```suggestion
ReactGA.initialize('UA-7529271-31');
```